src/tls_wrappers/openssl/un_negotiate.c

```diff
@@ -293,8 +293,12 @@ int verify_certificate(int preverify, X509_STORE_CTX *ctx)
 {
 #endif
 
+/*
+* This code allows you to use command: openssl x509 -in /tmp/cert.der -inform der -text -noout
```

Comment:
Change to `This code allows you to use command "openssl x509 -in /tmp/cert.der -inform der -text -noout" to dump the content of TLS certificate with evidence extension.`

Reply:
This is indeed more accurate！